extras/capitanodoc/markdown.ts

```diff
@@ -0,0 +1,90 @@
+import * as _ from 'lodash';
+import * as ent from 'ent';
+import * as utils from './utils';
+import { Document, Category, Command } from './doc-types';
+
+export function renderCommand(command: Command) {
+	let result = `\
+## ${ent.encode(command.signature)}
```

Comment:
It is weird. I played with common tags briefly but it seemed to get confused by some of the indentation here. Not sure why, but I've just simplified these to single line strings and explicit newlines instead, since they're only short.